[2025-06-08T10:24:00.476+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_data_pipeline.run_kafka_producer scheduled__2025-06-08T10:22:00+00:00 [queued]>
[2025-06-08T10:24:00.481+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_data_pipeline.run_kafka_producer scheduled__2025-06-08T10:22:00+00:00 [queued]>
[2025-06-08T10:24:00.481+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-08T10:24:00.487+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_kafka_producer> on 2025-06-08 10:22:00+00:00
[2025-06-08T10:24:00.491+0000] {standard_task_runner.py:60} INFO - Started process 8442 to run task
[2025-06-08T10:24:00.500+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'kafka_data_pipeline', 'run_kafka_producer', 'scheduled__2025-06-08T10:22:00+00:00', '--job-id', '348', '--raw', '--subdir', 'DAGS_FOLDER/kafka_data_pipeline.py', '--cfg-path', '/tmp/tmp_5mfi28l']
[2025-06-08T10:24:00.501+0000] {standard_task_runner.py:88} INFO - Job 348: Subtask run_kafka_producer
[2025-06-08T10:24:00.551+0000] {task_command.py:423} INFO - Running <TaskInstance: kafka_data_pipeline.run_kafka_producer scheduled__2025-06-08T10:22:00+00:00 [running]> on host 677411ab3888
[2025-06-08T10:24:00.604+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_data_pipeline' AIRFLOW_CTX_TASK_ID='run_kafka_producer' AIRFLOW_CTX_EXECUTION_DATE='2025-06-08T10:22:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-08T10:22:00+00:00'
[2025-06-08T10:24:00.605+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-08T10:24:00.606+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/ecom_etl/kafka_scripts/producer.py']
[2025-06-08T10:24:00.617+0000] {subprocess.py:86} INFO - Output:
[2025-06-08T10:24:02.910+0000] {subprocess.py:93} INFO - Error: NoBrokersAvailable
[2025-06-08T10:24:02.973+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-08T10:24:03.010+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=kafka_data_pipeline, task_id=run_kafka_producer, execution_date=20250608T102200, start_date=20250608T102400, end_date=20250608T102403
[2025-06-08T10:24:03.053+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-08T10:24:03.096+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
